hudi-cli/src/test/java/org/apache/hudi/cli/testutils/HoodieTestReplaceCommitMetadatGenerator.java

```diff
@@ -0,0 +1,74 @@
+package org.apache.hudi.cli.testutils;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hudi.common.model.HoodieReplaceCommitMetadata;
+import org.apache.hudi.common.model.HoodieWriteStat;
+import org.apache.hudi.common.table.timeline.HoodieTimeline;
+import org.apache.hudi.common.testutils.FileCreateUtils;
+import org.apache.hudi.common.util.Option;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.UUID;
+
+import static org.apache.hudi.common.testutils.FileCreateUtils.baseFileName;
+import static org.apache.hudi.common.util.CollectionUtils.createImmutableList;
+
+public class HoodieTestReplaceCommitMetadatGenerator extends HoodieTestCommitMetadataGenerator{
+    public static void createReplaceCommitFileWithMetadata(String basePath, String commitTime, Configuration configuration,
+                                                           Option<Integer> writes, Option<Integer> updates) throws Exception {
+        createReplaceCommitFileWithMetadata(basePath, commitTime, configuration, UUID.randomUUID().toString(),
+                UUID.randomUUID().toString(), writes, updates);
+    }
+
+    private static void createReplaceCommitFileWithMetadata(String basePath, String commitTime, Configuration configuration,
+                                                            String fileId1, String fileId2, Option<Integer> writes,
+                                                            Option<Integer> updates) throws Exception {
+        List<String> commitFileNames = Arrays.asList(HoodieTimeline.makeCommitFileName(commitTime),
```

Comment:
Can we reuse replace commit generator from other places? HoodieTestTable for example?